worlds/alttp/docs/multiworld_en.md

```diff
@@ -3,13 +3,16 @@
 ## Required Software
 
 - [Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases). Make sure to check the box for `SNI Client - A Link to the Past Patch Setup`
+- [SNI](https://github.com/alttpo/sni/releases). This is automatically included with your Archipelago installation above,
+and *IS NOT COMPATIBLE WITH (Q)Usb2Snes*
 - Hardware or software capable of loading and playing SNES ROM files
     - An emulator capable of connecting to SNI
       ([snes9x rr](https://github.com/gocha/snes9x-rr/releases),
        [BizHawk](http://tasvideos.org/BizHawk.html), or
        [RetroArch](https://retroarch.com?page=platforms) 1.10.1 or newer). Or,
     - An SD2SNES, [FXPak Pro](https://krikzz.com/store/home/54-fxpak-pro.html), or other compatible hardware. **note: 
-modded SNES minis are currently not supported by SNI**
+modded SNES minis are currently not supported by SNI. Some users have claimed success with QUsb2Snes for this system
+but support is not guaranteed.**
```

Comment:
```suggestion
modded SNES minis are currently not supported by SNI. Some users have claimed success with QUsb2Snes for this system,
it is not supported.**
```
I'm not sure whether the semantics of this suggestion make it less correct than your original proposal, but the impression I get is that QUsb2Snes is emphatically *not* supported, and I think the phrasing of "support is not guaranteed" could undermine the goal of getting people to please stop trying.

Reply:
For the sake of English grammar, line 15 should be something along the lines of 'but it is not supported', since either a conjunction or sentence separation is required.